projects/pgai/tests/vectorizer/conftest.py

```diff
@@ -113,3 +119,50 @@ def postgres_container(
     load_openai_key: bool = params.get("load_openai_key", True)  # type: ignore
 
     return postgres_container_manager(load_openai_key=load_openai_key)  # type: ignore
+
+
+class ReverseProxy:
```

Comment:
Maybe this sound stupid, but I think it would be better to name this ReverseProxyConfig as this is not really the proxy itself, this just carries the config, otherwise I would expect this class to provide a method "run" to execute the proxy. 🙂

Reply:
Renamed to `ReverseProxyAddon` which is what it is actually. Does it make sense?